plugin/pkg/admission/resourcequota/controller.go

```diff
@@ -105,7 +105,7 @@ func NewQuotaEvaluator(quotaAccessor QuotaAccessor, registry quota.Registry, loc
 
 		registry: registry,
 
-		queue:      workqueue.New(),
+		queue:      workqueue.NewNamed("admission_quota_controller"),
```

Comment:
I realize this is pkg/admission and not pkg/controller, but would you want to do admission_controller_resourcequota?